sparkles/yoshi.py

```diff
@@ -53,15 +53,15 @@ def get_yoshi_params_from_ocat(obsid, obs_date=None, web_ocat=True):
     if ocat["z_sim"] is not numpy.ma.masked:
         targ["sim_offset"] = ocat["z_sim"] * 397.7225924607
 
-    # Could use get_target_aimpoint but that needs icxc if not on HEAD
-    # and we don't need to care that much for future observations.
-    aimpoints = {
-        "ACIS-I": (970.0, 975.0, 3),
-        "ACIS-S": (210.0, 520.0, 7),
-        "HRC-S": (2195.0, 8915.0, 2),
-        "HRC-I": (7590.0, 7745.0, 0),
-    }
-    chip_x, chip_y, chip_id = aimpoints[ocat["instr"]]
+    # The zero offset aimpoint table starts at cycle 15.
+    # For ancient obsids, just use the earliest in the table.
+    cycle = np.max([15, int(ocat['prop_cycle'])])
```

Comment:
I wasn't exactly sure what to do for obsids that pre-date the zero-offset aimpoint table process, so I just set them to have cycle=15 for now.  I don't see this as a particularly useful case for to worry about for this code.

Reply:
Good enough!